airflow/providers/apache/beam/triggers/beam.py

```diff
@@ -233,6 +235,22 @@ async def run(self) -> AsyncIterator[TriggerEvent]:  # type: ignore[override]
                 if is_running:
                     await asyncio.sleep(self.poll_sleep)
         try:
+            # Creating a new event loop to manage I/O operations asynchronously
```

Comment:
I am not sure (I am not that experienced in asyncio)  if that one is correct. the docs https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.get_event_loop  
suggest that if get_event_loop() is run in the coroutine, it will not create a new event loop - it will return the current loop, and suggestion is to use `get_running_loop` instead, so the comment is quite a bit misleading. 

Can you explain what's the expected behaviour here please ?

Reply:
BTW. I see it was copied from the previous implementation but it might be good to clarify and possibly fix it to use get_running_loop() (and fix the comment) if that's what is expected.

Reply:
I also feel like this might not be necessary if we already run it in the trigger. @MaksYermak Could you please check whether we can get rid of the event_loop? Thanks!

Reply:
@potiuk @Lee-W I have checked. We can't get rid of `event_loop` here, because as I understand it's only one option to execute a synchronous method in async code. Also we can't change this method to async, because google does not have async client for this method. But we can change `get_event_loop` to `get_running_loop` for improving code quality. I will do it in the next commit.

Reply:
Thanks for checking it.